.dockstore.yml

```diff
@@ -167,14 +167,34 @@ workflows:
          - ah_var_store
        tags:
          - /.*/
+   - name: GvsMergeAndRescoreVDSes
```

Comment:
For a separate ticket: update AoU instructions to describe how we're going to use this for Foxtrot.